### PR TITLE
HOTFIX | Prepend FORCE_SCRIPT to ignored request paths

### PR DESCRIPTION
**Pull request by a-rmz**
## Description :sparkles:
* Prepend the `FORCE_SCRIPT` to the ignored paths for logging

### CODE REVIEW

**Review by haavikko — COMMENTED** [review]
forgot to click submit earlier, here are my comments!

File: berth_reservations/settings.py [hunk 1]
```diff
@@ -77,11 +77,20 @@
...
+    "/static",
+)
+
 if env("CSRF_TRUSTED_ORIGINS"):
     CSRF_TRUSTED_ORIGINS = env.list("CSRF_TRUSTED_ORIGINS")
 
 if env.str("FORCE_SCRIPT_NAME"):
     FORCE_SCRIPT_NAME = env.str("FORCE_SCRIPT_NAME")
+    REQUEST_LOGGER_IGNORE_PATHS = [
+        f"{FORCE_SCRIPT_NAME}/{path}" for path in REQUEST_LOGGER_IGNORE_PATHS
```
[the remark this answers is not in the document]
Comment (a-rmz) [reply berth_reservations/settings.py:92]: Good catch!

File: berth_reservations/settings.py [hunk 2]
```diff
@@ -77,11 +77,20 @@
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
 USE_X_FORWARDED_HOST = env.bool("USE_X_FORWARDED_HOST")
 
+REQUEST_LOGGER_IGNORE_PATHS = (
```
[the remark this answers is not in the document]
Comment (a-rmz) [reply berth_reservations/settings.py:80]: Right! Both work since it checks on an iterable, but I'll change this one to tuple as well for conistency.

**Review by haavikko — APPROVED** [review]
full speed ahead

**Pull request merged**